modules/luigi/phrases/greeting.yml

```diff
@@ -7,9 +7,9 @@ Q:                                          # possible questions
 - "you look like you could use some ice cream! do you want some?"
 A:                                          # answers
   SUCCESS:
-  - "i knew it - everyone loves ice cream! well, what can I get you?"
-  - "that's the spirit! what kind of ice cream do you want?"
-  - "you have chosen ... wisely. what can i get you adventurer?"
+  - "i knew it - everyone loves ice cream!"
+  - "that's the spirit!"
+  - "you have chosen ... wisely."
```

Comment:
I am not sure about deleting followup questions. :thinking: But in any case it is fine for me.

Reply:
I had to delete them because I changed the conversation flow a little and the question about what the customer wants is asked only later.